README.md

```diff
@@ -49,6 +49,7 @@ You can create and manage connections to graph databases using this feature. Con
   - **Using proxy server:** Check this box if using a proxy endpoint.
   - **Graph connection URL:** Provide the endpoint for the graph database
   - **AWS IAM Auth Enabled:** Check this box if connecting to Amazon Neptune using IAM Auth and SigV4 signed requests
+  - **Service Type:** Choose the service type
```

Comment:
Let's explain inline (so it is easier for folks to understand)  - choose service time `neptune-db` for Neptune database, `neptune-graph` for Neptune Analytics or empty when not using IAM authentication. ?